pandas/core/arrays/boolean.py

```diff
@@ -740,6 +742,171 @@ def boolean_arithmetic_method(self, other):
         return set_function_name(boolean_arithmetic_method, name, cls)
 
 
+def kleene_or(
```

Comment:
would these be better in a separate module (in case we decide to re-use them) and this becomes less cluttered